mslib/msui/mscolab.py

```diff
@@ -694,6 +697,30 @@ def after_login(self, emailid, url, r):
             self.signal_login_mscolab.emit(self.mscolab_server_url, self.token)
 
     def fetch_gravatar(self, refresh=False):
```

Comment:
This function should probably be renamed, gravatar is a specific service and not a general name for profile images.